src/main/kotlin/eu/europa/ec/eudi/sdjwt/Specs.kt

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2023 European Commission
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package eu.europa.ec.eudi.sdjwt
+
+/**
+ * [Selective Disclosure for JWTs (SD-JWT)](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/)
+ */
+@Suppress("UNUSED")
+object SdJwtSpec {
```

Comment:
@babisRoutis Would it make sense to further segregate the constants inside nested objects and remove the prefixes? For instance `SdJwtSpec.CLAIM_SD` would become `SdJwtSpec.Claims.SD`.

Reply:
No